framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1338,33 +1176,23 @@ wifi_manager_result_e _handler_on_softap_state(_wifimgr_msg_s *msg)
 		WIFIMGR_CHECK_RESULT(_wifimgr_scan((wifi_manager_ap_config_s *)msg->param), "fail scan", WIFI_MANAGER_FAIL);
 		WIFIMGR_STORE_PREV_STATE;
 		WIFIMGR_SET_STATE(WIFIMGR_SCANNING);
+#ifdef CONFIG_WIFIMGR_DISABLE_DHCPS
 	} else if (msg->event == EVT_JOINED) {
-#ifndef CONFIG_WIFIMGR_DISABLE_DHCPS
-		/* wifi manager passes the callback after the dhcp server gives a station an IP address*/
-	} else if (msg->event == EVT_DHCPD_GET_IP) {
-		dhcp_node_s *node = (dhcp_node_s *)msg->param;
-		if (node != NULL) {
-			ndbg("[WM] IP: %d:%d:%d:%d\n",
-				  ((char *)&(node->ipaddr))[0],
-				  ((char *)&(node->ipaddr))[1],
-				  ((char *)&(node->ipaddr))[2],
-				  ((char *)&(node->ipaddr))[3]);
-			ndbg("[WM] MAC: %02x:%02x:%02x:%02x:%02x:%02x\n",
-				  node->macaddr[0], node->macaddr[1],
-				  node->macaddr[2], node->macaddr[3],
-				  node->macaddr[4], node->macaddr[5]);
-
-			int is_exist = _dhcps_add_node(node);
-
-			if (is_exist == DHCP_EXIST) {
-				return WIFI_MANAGER_SUCCESS;
-			}
+#else
+	/* wifi manager passes the callback after the dhcp server gives a station an IP address*/
+	} else if (msg->event == EVT_DHCPS_ASSIGN_IP) {
+		_dhcp_status_e is_exist = _dhcps_add_node((dhcp_node_s *)msg->param);
+		if (is_exist == DHCP_EXIST) {
+			return WIFI_MANAGER_SUCCESS;
 		}
 #endif
-		WIFIMGR_INC_NUM_CLIENT;
+		_dhcps_inc_num();
```

Comment:
I think '_' in front of a function name means a static function. so how about removing _ in dhcp functions

Reply:
That's good point.
Applied your suggestion.
Thanks for the comment.